### PR TITLE
Adds monitoring a folder based on its number of files

### PR DESCRIPTION
**Pull request by iramello**

### CODE REVIEW

File: bin/riemann-dir-files [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @dir = opts.fetch(:directory)
+    @service_prefix = opts.fetch(:service_prefix)
+    @warning = opts.fetch(:warning, nil)
+    @critical = opts.fetch(:critical, nil)
+    @alert_on_missing = opts.fetch(:alert_on_missing)
+  end
+
+  def tick
+    if Dir.exists?(@dir)
+      metric = `ls -l '#{@dir}'|grep "^-"|wc -l`.to_i
```
Comment (aphyr) [inline bin/riemann-dir-files]: You can write `@dir.entries.size - 2` instead.

File: bin/riemann-dir-files [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+# Gets the number of files present on a directory and submits it to riemann
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+
+class Riemann::Tools::DirFiles
```
Comment (aphyr) [inline bin/riemann-dir-files]: Would it make sense to call this FileCount instead?

File: bin/riemann-dir-files (hunk 1 above)
Comment (iramello) [inline bin/riemann-dir-files]: Good observation! Will change it on a commit below.

File: bin/riemann-dir-files (hunk 2 above)
Comment (iramello) [inline bin/riemann-dir-files]: I just named it DirFiles to keep a naming convention with the existent "DirSpace" class (riemann-dir-space), as it's a similar function over a single folder with a different purpose of course. We can rename it if you wish, it's your call. Just reply me here what you prefer and I will change the class name (also the filename).

Comment (aphyr) [inline bin/riemann-dir-files]: Yeah, I could see "DirFiles" meaning any number of things, so I'd include the phrase "Count" somewhere in the name. FileCount, DirCount, DirFilesCount, something like that. :)

Comment (iramello) [inline bin/riemann-dir-files]: Done.

**Comment by aphyr** [comment]
Terrific, thank you very much, @iramello. :)

**Comment by aphyr** [comment]
I've added you to the [contributor's list](http://riemann.io/thanks.html) as well. :D

**Comment by iramello** [comment]
haha thanks aphyr! :)

**Pull request merged**